djangosaml2idp/views.py

```diff
@@ -93,7 +110,37 @@ def set_sp(self, sp_entity_id):
         try:
             self.sp['config'] = settings.SAML_IDP_SPCONFIG[sp_entity_id]
         except KeyError:
-            raise ImproperlyConfigured("No config for SP {} defined in SAML_IDP_SPCONFIG".format(sp_entity_id))
+            raise ImproperlyConfigured(_("No config for SP {} defined in SAML_IDP_SPCONFIG").format(sp_entity_id))
+
+    def build_response_arguments(self, req_info):
```

Comment:
For this and the next 2 methods, the error handling won't work because we're not doing anything with the return value. I'd say we should raise the exceptions in the methods, and then wrap all the calls to the methods in an error handler that would then return the handle_error view

Reply:
Ok, If I remeber well these could be legacy code that I adapted with internazionalization, nothing else. I agree